src/shuffle.ts

```diff
@@ -59,7 +63,7 @@ function randomRange(min: number, max: number): number {
  * @param {Array} array - Any type of array
  * @return {Array} - The sorted array
  */
-export function shuffle(array: any[]): any[] {
+export function shuffle(array) {
```

Comment:
We need to keep the input type (the return type is inferred, so not needed)
```suggestion
export function shuffle(array: any[]) {
```